librz/include/sflib/freebsd-x86-32/sflib.h

```diff
@@ -0,0 +1,293 @@
+// SPDX-FileCopyrightText: 2004  Philippe Biondi <phil@secdev.org>
```

Comment:
Too many spaces, remove duplicate.

Reply:
@XVilka I can see the too many spaces issue, but I am not sure about the duplicate. Do you mean the copyright part here as a duplicate to the corresponding reference in the `.reuse/dep5` (i.e the copyright of the `librz/include/sflib/*` file)? Thanks in advance!